bin/run_tertiary_workflow.sh

```diff
@@ -88,19 +69,28 @@ if [ -n "$batch_field" ]; then
 fi
     
 sub_in_params 'cell_type_field' "$cell_type_field"
-sub_in_params 'batch_variable' $batch_field
-sub_in_params 'representation' $representation
-
-run_galaxy_workflow.py -C $GALAXY_CRED_FILE \
-                       -i $inputs_yaml \
-                       -o $WORKDIR \
-                       -W $flavor_dir/scanpy_clustering_workflow.json \
-                       -P $parameters_yaml \
```

Comment:
I will look into `parameters.yaml`

Reply:
As far as I understand `parameters.yaml` file is the params per step and addition of `cell_type_field`, `batch_variable` and `representation`

We have set parameters per process and three additional params, we pass them as NF param so this is not needed

Please correct me if i am wrong

Reply:
@anilthanki please check that you are happy with  https://github.com/ebi-gene-expression-group/scxa-workflows/pull/38/commits/3a8da3675f5eef98a9e317902325397ea699fc23